catboost/python-package/ut/medium/canondata/result.json

```diff
@@ -16,11 +32,11 @@
         "uri": "file://test.test_class_names_CPU_/predictions.txt"
     },
     "test.test_class_weights[CPU]": {
-        "checksum": "9f46bfee564b816cdbee529aeb539f36",
```

Comment:
Remove all changes in not related to pr tests, please

Reply:
It seems, I have (finally!) managed to remove excessive changes. Yet, it probably is based on changes in main repository, so it likely wouldn't last long.